package.json

```diff
@@ -5,7 +5,11 @@
   "main": "lib/Slider",
   "jsnext:main": "src/Slider.jsx",
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "npm run clean && babel src -d lib",
+    "postbuild": "npm run build:css",
```

Comment:
I still prefer to have:
```
"build:js": "babel src -d lib",
```
as a separate script and then have `"build"` be `"npm run build:js && npm run build:css"`. The CSS step is not a postbuild, it's part of the build itself.